webpack/assets/javascripts/react_app/components/bookmarks/index.js

```diff
@@ -45,12 +41,17 @@ class BookmarkContainer extends React.Component {
         <Dropdown.Toggle />
         <Dropdown.Menu>
           <MenuItem header>{__('Saved Bookmarks')}</MenuItem>
-          {getLoadingStatus(errors, bookmarks) === STATUS.PENDING && (
-            <Loader spinnerSize="xs" status={STATUS.PENDING} />
+          {status === STATUS.PENDING && (
+            <MenuItem className='loader-root'>
```

Comment:
I think we should add the spinner as a direct child element to the `<li>` in the DOM and drop the `<a>` around it.